cdap-ui/app/cdap/components/AbstractWidget/FormInputs/Select/index.tsx

```diff
@@ -21,35 +21,59 @@ import { IWidgetProps } from 'components/AbstractWidget';
 import { objectQuery } from 'services/helpers';
 import { WIDGET_PROPTYPES } from 'components/AbstractWidget/constants';
 import withStyles from '@material-ui/core/styles/withStyles';
-
+import { blue } from 'components/ThemeWrapper/colors';
 const CustomizedInput = withStyles(() => {
   return {
     input: {
       padding: '7px 18px 7px 12px',
       '&:focus': {
         backgroundColor: 'transparent',
+        outline: `1px solid ${blue[100]}`,
```

Comment:
we can use `theme` in `withStyles` I believe.

`withStyles((theme))`

Reply:
https://github.com/cdapio/cdap/pull/12450#discussion_r455414457